client/components/gradeForm.jsx

```diff
@@ -48,22 +48,34 @@ class GradeForm extends React.Component {
   render() {
     return (
       <form onSubmit={this.handleSubmit}>
+        <div className='row'>
+          <div className='m-1'>
+            <div className='mb-3'><i className='fas fa-user'></i></div>
+            <div className='mb-3'><i className='fas fa-list'></i></div>
+            <div className='mb-2'><i className='fas fa-graduation-cap'></i></div>
+          </div>
+          <div>
+            <div className='mb-2'><input type='name' placeholder='Name' onChange={this.handleName} value={this.state.name}></input></div>
```

Comment:
Check out the form control and form group classes.

https://getbootstrap.com/docs/4.4/components/forms/